src/arduino/comms/Handshake.cpp

```diff
@@ -0,0 +1,40 @@
+#include "handshake.h"
+
+/**
+ * Returns pressure in PSI
+ *
+ * LOGIC:
+ * 1. Read sensor value
+ * 2. Convert sensor value to voltage
+ * 3. Convert voltage to pressure
+ * 4. Set pressure
+ * 5. Return pressure
+ */
+void Handshake::toggle()
+{
+  status = !status;
+  digitalWrite(pin, status);
+}
+
+void Handshake::setHigh()
+{
+  status = true;
+  digitalWrite(pin, status);
+}
+
+void Handshake::setLow()
+{
+  status = false;
+  digitalWrite(pin, status);
+}
+
+void Handshake::setStatus(bool status)
+{
+  this->status = status;
+  digitalWrite(pin, status);
+}
```

Comment:
Is there any real benefit to having `setHigh`, `setLow`, and `setStatus` here, rather than just having `setStatus`? It currently seems a bit repetitive, and it doesn't look like you use the first two methods anywhere.

Reply:
I'm just making exhaustive list of functions theres no real reason to have one over the other. it's just more explicit if we used setLow and setHigh.

Reply:
I agree readability is important, so here is another way to go about this. Arduino actually already comes with `#define` macros for `HIGH` and `LOW` to represent pin states. So we could just do `handshake.setStatus(HIGH)` without compromising readability but still being able to shorten the class definition

Reply:
Tbh that isn't my priority plus having a longer definition doesn't hurt so I really don't see your point. If you are trying to debate sacrificing readability over preference of shorter code definitions, then I will not worry about this at this time.